subsys/bluetooth/shell/hci.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __HCI_H_
```

Comment:
All header guards in Zephyr were recently changed to be constructed out of the full path. This file's should then be `ZEPHYR_SUBSYS_BLUETOOTH_SHELL_HCI_H_`

Reply:
Actually, does this file even need a guard? Isn't it completely internal?

Reply:
Agree will remove it.